mainStyle.css

```diff
@@ -3,7 +3,7 @@
 
 
 *{
-    font-family: 'cairo' , sans-serif;
+    font-family: 'poppins', sans-serif;
```

Comment:
The font family 'poppins' may not be viewed by some browsers  consequently I would suggest you use the 
@import url('https://fonts.googleapis.com/css2?family=Mukta:wght@800&family=Poppins&family=Roboto&family=Ubuntu&display=swap');
line to add the font to the current project